### ISSUE TITLE
Change Request Module: Different GeoObject options are showing for lower case vs upper case search letters

### ISSUE THREAD

**Issue by CarlyNel** [terraframe/geoprism-registry#453]
**List the CGR information**
- CGR version: 11 and 12 snapshot
- CGR instance (e.g. a URL): staging

**Describe the bug (clear and concise)**
When submitting a change request via the change request module, if I type a lower case a or upper case A (or any other letter) in the GeoObject field, the drop down menu options that appear are different even though I'm typing in the same letter.

**To Reproduce**
1. As a MOHA-RA go to the change request module
2. Select your GeoObject Type
3. Type a letter (upper case and then lower case) into the search field for the GeoObject
3. See error: different options displayed despite typing in the same letter

**Expected behavior (clear and concise)**
The same search results should be showing for the same letter - irrespective of whether I type the letter in upper or lower case.

**Screenshots**
![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/58c81de5-5150-44f1-9922-2178dcb51f2e)

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/34a802f9-ac34-493b-9cb4-121dac6cf0b0)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: 81.0.2 (64-bit)

**Additional context (if any)**
This info was taken from CHAI's recent testing [feedback](https://docs.google.com/spreadsheets/d/1VD9sTHAx5CukHyjTEH9--9GjVM-oLpJmpbw1AOmFTHw/edit#gid=1752608915). Line 36.

**Comment by SteeveEbener** [comment]
Linked if not included in ticket #341

**Comment by CarlyNel** [comment]
I have just looked at this search function within the Explorer (since the submit functionality has been removed from the change request module). It's much better to how it was originally.

There now appears to be consistency:
1) Searching in lower case or upper case returns the same results
2) The search function searches based on the first letter of the geo-object, and does not include geo-objects who have the searched-for letter in their name. See screenshots below.

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/3af15796-141e-4653-b46c-df9a57f98af5)

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/aceb0835-34db-4eab-9386-962c05778bad)

Side note: The search function also picks up geo-objects where part of the geo-object's name starts with the letter that's being searched for. Example below.

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/590ca4ad-b1bb-4624-9682-3ce7af8886a3)
